src/crawlers.js

```diff
@@ -1,105 +1,187 @@
 import _ from 'underscore';
 
+import { checkParamOrThrow } from './utils';
+
 export const BASE_PATH = '/v1';
 
 export default {
     listCrawlers: (requestPromise, options) => {
-        if (!options.userId) {
-            throw new Error('Missing required parameter: userId');
-        }
-        if (!options.token) {
-            throw new Error('Missing required parameter: token');
-        }
+        const { userId, token } = options;
+
+        checkParamOrThrow(userId, 'userId', 'String');
+        checkParamOrThrow(token, 'token', 'String');
 
         const queryString = _.pick(options, 'token', 'offset', 'query', 'desc');
 
         return requestPromise({
```

Comment:
We should encapsulate returned array in `items` parameter for future compatibility with pagination implementation:

```
{ 
    items: [ crawler1, crawler2, ... ],
}
```

Because we will have to add info about pagination which is received in headers - see https://www.apifier.com/api-reference#/introduction/pagination

So we will be then returning something like:

```
{
    items: [ crawler1, crawler2, ... ],
    count: 20,
    totalCount: 50,
    ...
}